common/db_test.go

```diff
@@ -118,6 +125,47 @@ func profilesMatch(j1 []ffmpeg.VideoProfile, j2 []ffmpeg.VideoProfile) bool {
 	return true
 }
 
+func TestDBOrchestrators(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+
+	orch := NewStubOrch("127.0.0.1:8936", "0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe")
+	err = dbh.UpdateOrchs(orch)
+
+	if err != nil {
+		t.Error("Unexpected error in update orchs: ", err)
+	}
+
+	orchs, err := dbh.SelectOrchs()
+	if err != nil || len(orchs) != 1 || orchs[0].ServiceURI != orch.ServiceURI {
+		t.Error("Unexpected error in update orchs: ", err)
+	}
+
+	orchUpdate := NewStubOrch("127.0.0.1:8937", "0xde0B295669a9FD93d5F28D9Ec85E40f4cb697BAe")
+	err = dbh.UpdateOrchs(orchUpdate)
+
+	if err != nil {
```

Comment:
Would you like to use the new `testify` library we've been using? - it should allow you to replace all these if blocks with a simple `assert.Nil(err)`. wdyt?

Reply:
Yessss!